### ISSUE TITLE
LightGaussian is here!!

### ISSUE THREAD

**Issue by LifeIsStrange** [aras-p/UnityGaussianSplatting#76]
Lightgaussian is significantly faster than the original gaussian splatting (and btw generates scenes ten times smaller)
https://lightgaussian.github.io/

**Comment by aras-p** [comment]
Yes, in the recent weeks there's been several dozen papers coming up with various improvements to the OG gaussian splatting paper. I suspect many more will come over the coming months. I don't plan to implement/work on any of them at the moment, thus closing this issue.
